src/encoded/types/modification.py

```diff
@@ -22,10 +22,19 @@ class Modification(Item):
     item_type = 'modification'
     schema = load_schema('encoded:schemas/modification.json')
     embedded_list = Item.embedded_list + lab_award_attribution_embed_list + [
+        # Construct linkTo
+        'constructs.name',
         'constructs.construct_type',
         'constructs.tags',
         'constructs.designed_to_target',
+
+        # GenomicRegion linkTo
         'modified_regions.aliases',
```

Comment:
is there a reason the aliases were  embedded?

Reply:
I'm not sure, I think this embed was there before I made changes.